src/DcrdClient/Values/SearchRawTransactionsResult.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections.Generic;
+
+namespace DcrdClient
```

Comment:
Please, follow "one class/interface - one file" rule, if it is not nested classes/interfaces.
Follow Microsoft naming-convensions for property names: they should be in CamelCase, i.e. "txid" should be "TxId", "amountin" should be "AmountIn", and so on. If you experience some problems with deserialisation, use ```JsonProperty``` attribute.